### DISCUSSION TITLE
User Dashboard Q: Is it possible to hide a specific app (forward proxt auth) from the dashboard?

### DISCUSSION THREAD

**Discussion by Lockszmith-GH** [goauthentik/authentik#8651] [Q&A] [answered]
I find the dashboard useful, reminds me of the OKTA solution at work.
Is it possible to hide an app, without revoking access?

The forward proxy app, isn't really useful - as it's a 'catchall' for the services that use it...

**Comment by BeryJu** [comment, marked as answer]
See https://goauthentik.io/docs/applications#hiding-applications

**Reply by Lockszmith-GH** [reply]
`blank://blank` - so simple

**Reply by Lockszmith-GH** [reply]
Just a note for anyone stumbling into this later: After chanigng the Launch URL and saving, I needed to logout and login again in order for the button to dissapear from the dashboard.
